tutorials/raw/plot_40_visualize_raw.py

```diff
@@ -192,3 +192,4 @@
 # .. LINKS
 #
 # .. _spectral density: https://en.wikipedia.org/wiki/Spectral_density
+
```

Comment:
you'll need to revert this before we can merge; it will fail flake8

Reply:
Yep, just wanted to ensure the tutorial would be rendered… or would that have happened anyway?

Reply:
Oh I see, it doesn't render anything now haha

Reply:
yeah, the triggering is determined by comparing the file against master.

What I should have said is "when you touch a tutorial to trigger it to render, a safe way to touch it is to just add/change a line break in a paragraph of text.  Adding blank lines in the middle of the document (e.g., in between paragraphs or between a paragraph and a code block) should also be safe.  Blank lines at the end make flake angry, and `XXX` comments will show up in the rendered tutorial (and so are also not a good choice).

Reply:
Thanks for the explanation, @drammock!